README.md

```diff
@@ -39,6 +39,7 @@ Though Scriptable has its own documentation, found [here](https://docs.scriptabl
   - Some good references for Javascript can be found on [w3schools](https://www.w3schools.com/js/js_intro.asp) or [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide). Good beginner courses can be found on [Codecademy](https://www.codecademy.com/learn/introduction-to-javascript) or [SoloLearn](https://www.sololearn.com/Course/JavaScript/?ref=app).
 - Access to an iOS device. 
 - iOS 14 or later. 
+- You have the [Scriptable app](https://apps.apple.com/us/app/scriptable/id1405459188) or [Testflight beta](https://testflight.apple.com/join/uN1vTqxk)
```

Comment:
This works fine, I'm gonna add a period to the end of your list item. I'm also going to try adding the Scriptable icon image. :)